jaxopt/_src/eq_qp.py

```diff
@@ -62,21 +67,33 @@ def optimality_fun(params, params_obj, params_eq):
   return optimality_fun
 
 
+@dataclass
+class EqualityConstrainedQPState:
```

Comment:
No, indeed. I am waiting for `linear_solve` solvers to return a Tuple whose second member without be infos on the algforithm (error, iter_num, etc... ). 

Not useful currently but I hope it will be in the future once linear solvers adopt the Jaxopt base.IterativeSolver interface.

Reply:
Let's delete it for the time being then.